phobos/blender/io/blender2phobos.py

```diff
@@ -430,11 +430,18 @@ def deriveLink(obj, objectlist=None, logging=True, errors=None):
                     annotations[k1] = {}
                 annotations[k1][k2] = v
 
+    root = obj
+    while sUtils.getEffectiveParent(root) is not None:
+        root = sUtils.getEffectiveParent(root)
+
+    # Exporting pose relative to root link because gazebo 9 ignores relative_to attribute
+
     return representation.Link(
         name=obj.get("link/name", obj.name),  # this is for backwards compatibility normally the linkname should be the object name
         visuals=visuals,
         collisions=collisions,
         inertial=inertial,
+        origin=deriveObjectPose(obj, effectiveparent=root),
```

Comment:
Are you sure this is necessary? The link origin is only used when the link has another origin then defined by the joint.
But seems like a valid change. Please test whether the URDF export is still valid after this change.

Reply:
You're right, this breaks the urdf export. Fixed that